lib/async.js

```diff
@@ -863,8 +863,17 @@
         function _next(q, tasks) {
             return function(){
                 workers -= 1;
+
```

Comment:
@megawac, I do not know what I guided when giving named. Renamed to `removed`. :smile: